sdgym/result_explorer/result_explorer.py

```diff
@@ -62,14 +113,17 @@ def load_synthetic_data(self, results_folder_name, dataset_name, synthesizer_nam
 
     def load_real_data(self, dataset_name):
         """Load the real data for a given dataset."""
-        if dataset_name not in DEFAULT_SINGLE_TABLE_DATASETS:
+        # Keep strict validation for single_table to preserve existing behavior
+        if (self.modality is None or self.modality == 'single_table') and (
+            dataset_name not in DEFAULT_SINGLE_TABLE_DATASETS
+        ):
```

Comment:
Should we validate the dataset for `multi-table` here as well?

Reply:
Isn't it possible for a valid dataset to not be part of DEFAULT_MULTI_TABLE_DATASETS?